Resources/Prototypes/Entities/Objects/Specific/Cargo/mail_bag.yml

```
@@ -0,0 +1,30 @@
+- type: entity
+  name: mail bag
+  id: MailBag
+  parent: BaseStorageItem
+  description: A bag for carrying mail around the station. Made from tanned space bear hide.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Cargo/mail_bag.rsi
+    state: icon
+  - type: Clothing
+    sprite: Objects/Specific/Cargo/mail_bag.rsi
+    quickEquip: false
+    slots:
+    - belt
+  - type: Item
+    size: Ginormous
+  - type: Storage
+    maxItemSize: Normal
+    grid:
+    - 0,0,4,3
+    quickInsert: true
+    areaInsert: true
+    whitelist:
+      components:
+        - Paper
+        - Delivery
+      tags:
+        - Document
+        - Paper
```

```suggestion
      components:
      - Paper
      - Delivery
      tags:
      - Document
      - Paper
```
[yaml convention](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions)